src/PhotoCatalog.WebAPI/PhotoCatalog.WebAPI/PhotoCatalog.Service/Services/ImageService.cs

```diff
@@ -236,5 +238,30 @@ public IEnumerable<string> GetAllFilesPaths()
             di.FillAllFilenames(_imageSettings.ImagesFolderName);
             return di.AllFilePaths;
         }
+
+        public async Task<ImageDTO> DeleteImage(string imagePath)
+        {
+            try
+            {
+                try
+                {
```

Comment:
Używasz łącznie 3 razy try-catch do tego samego xd
Trochę to bez sensu. Raz wystarczy. Ewentualnie w warstwie Middleware można zrobić przechwytywanie wyjątków